src/librustc_driver/pretty.rs

```diff
@@ -199,16 +199,16 @@ impl PpSourceMode {
         }
     }
     fn call_with_pp_support_hir<'tcx, A, F>(&self,
-                                               sess: &'tcx Session,
-                                               cstore: &'tcx CStore,
-                                               hir_map: &hir_map::Map<'tcx>,
-                                               analysis: &ty::CrateAnalysis,
-                                               resolutions: &Resolutions,
-                                               arenas: &'tcx AllArenas<'tcx>,
-                                               output_filenames: &OutputFilenames,
-                                               id: &str,
-                                               f: F)
-                                               -> A
+                                            sess: &'tcx Session,
+                                            cstore: &'tcx CStore,
+                                            hir_map: &hir_map::Map<'tcx>,
+                                            analysis: &ty::CrateAnalysis,
+                                            resolutions: &Resolutions,
+                                            arenas: &'tcx AllArenas<'tcx>,
+                                            output_filenames: &OutputFilenames,
+                                            id: &str,
+                                            f: F)
+                                            -> A
```

Comment:
When encountering these I usually change them to be
```
fn foo(
    &self,
    bar,
) -> A {
}
```

That way we won't have to worry about the function name changing length or adding/removing type args/lifetimes requiring reindenting of the arguments.